api-gateway-config/scripts/lua/lib/redis.lua

```diff
@@ -169,6 +212,59 @@ function _M.deleteResource(red, key, field)
   end
 end
 
+-----------------------------
+---------- Tenants ----------
+-----------------------------
+
+--- Add tenant to redis
+-- @param red Redis client instance
+-- @param id id of tenant
+-- @param tenantObj the tenant to add
+function _M.addTenant(red, id, tenantObj)
+  local ok, err = red:hset("tenants", id, tenantObj)
+  if not ok then
+    request.err(500, utils.concatStrings({"Failed adding tenant to redis: ", err}))
```

Comment:
Same comment here as for the APIs--make things generic, but also log the message to someone can see in the logs what went wrong.